### ISSUE TITLE
modify in-flight messages

### ISSUE THREAD

**Issue by nathanielhudson** [FrazerSeymour/flip#18]

**Comment by FrazerSeymour** [comment]
What's this about?

**Comment by nathanielhudson** [comment]
Such that multiple burritos collapse into a single message of many burritos.

**Comment by FrazerSeymour** [comment]
Riiiiiight. :+1:

**Comment by nathanielhudson** [comment]
Any reason we're not using https://developer.mozilla.org/en/docs/Web/API/notification ? Seems to work awesome on OSX.

**Comment by FrazerSeymour** [comment]
Looks like that's what Slack uses, so go for it. I just grabbed the first one I found.

**Comment by nathanielhudson** [comment]
can you checkout the 18-messages branch and let me know if the notification and burrito stacking work correctly on your system? It mostly works here, but there's some strangeness (messages don't stack right after the first 6). I think it's OSX related, and might need some special case code.

**Comment by FrazerSeymour** [comment]
So, it doesn't stop at 6, but it doesn't update the in-flight notification either. It just creates a new notification with the right amount of :burrito:s.

**Comment by FrazerSeymour** [comment]
That _could_ be a byproduct of my janky WM setup.

**Comment by nathanielhudson** [comment]
Hmmm. I'm not quite sure what to do in that case. I might implement a client-side notification accumulator that we can turn on for some platforms then.

**Comment by FrazerSeymour** [comment]
I mean it uses libnotify, and I've updated in-flight messages before, but who knows. Might be worth testing it on Windows too.

**Comment by nathanielhudson** [comment]
Yeah - the way it's supposed to work as per the api spec and the electron docs is you send two messages with the same tag and the new will overwrite the old. In the real world it seems to be messier - probably worth testing on windows, like you said. I suspect we'll just have to cheat for uncooperative platforms...

**Comment by FrazerSeymour** [comment]
So mikaelbr/node-notifier uses the same API as julienXX/terminal-notifier, which has a `group` option for this purpose. Would that work I wonder? mikaelbr/node-notifier seemed to do cross-platform okay. I can test it on my laptop later and report back.
